src/Hazelcast.Net/DistributedObjects/Impl/HReplicatedMap.cs

```diff
@@ -39,13 +39,16 @@ public HReplicatedMap(string name, DistributedObjectFactory factory, Cluster clu
         }
 
         public Task<TValue> PutAsync(TKey key, TValue value)
-            => PutAsync(key, value, TimeToLive.InfiniteTimeSpan);
+            => PutAsync(key, value, TimeToLive.Infinite);
 
         public async Task<TValue> PutAsync(TKey key, TValue value, TimeSpan timeToLive)
```

Comment:
The default value is zero so this can be simplified as 
`public async Task<TValue> PutAsync(TKey key, TValue value, TimeSpan timeToLive = default)`
and remove the above call 
```
       public Task<TValue> PutAsync(TKey key, TValue value)
            => PutAsync(key, value, TimeToLive.InfiniteTimeSpan);	            => PutAsync(key, value, TimeToLive.Infinite);

```

Reply:
ok